news/2017-10-09-halloween-fanart-contest.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "Halloween 2017 Fanart Contest"
+date: 2017-10-09 06:00:00 +0000
+---
+
+Spooks, chills and thrills abound! Halloween is drawing close again, and you know what that means - a fanart contest! Two hectic weeks to pen any of the osu! mascots (or community mascots) in a delightfully spooky Halloween setting, for a chance at getting your work added to the game as a seasonal background!
+
+[![](https://assets.ppy.sh/contests/56/header.jpg)](https://osu.ppy.sh/community/contests/56)
+
+Halloween's quickly becoming something of an _osu!_ staple as far as seasonal events go, what with the [incredible showing](https://osu.ppy.sh/home/news/2016-10-29-halloween-cometh) from last year's spookiness.
```

Comment:
the word `what` is between `go,` and `with` ... is that supposed to be there?

Reply:
what with
phrase of what
1.
because of (used typically to introduce several causes of something).
"what with the drought and the neglect, the garden is in a sad condition"